ONBOARDING.md

```diff
@@ -28,6 +29,8 @@ invalid and are superseded by the TSC Charter.
 * Remove them from the `@nodejs-private/security-tsc` team.
 * Remove them from the `@nodejs-private` org unless they are members for a
   reason other than TSC membership.
+* Remove them from the `@pkgjs` org unless they are members for a
```

Comment:
* Remove them from the `@pkgjs/node-tsc` team
* Remove them from the `@pkgjs` org unless they are members for a